src/imp/schannel.rs

```diff
@@ -91,14 +91,11 @@ impl Identity {
             }
         };
         
-        let mut store = CertStore::open_current_user("RustMy").unwrap();
+        let mut store = CertStore::open_current_user("RustMy")?;
```

Comment:
You should create a new store name here. Don't want to override the above provider store.